sumo-search.gemspec

```diff
@@ -14,8 +14,8 @@ Gem::Specification.new do |gem|
   gem.name          = 'sumo-search'
   gem.require_paths = %w(lib)
   gem.version       = Sumo::VERSION
-  gem.add_dependency 'excon', '>= 0.32'
-  gem.add_dependency 'clamp', '>= 0.6.3'
+  gem.add_dependency 'excon', '~> 0.45.3'
```

Comment:
`>= 0.32` is scary, plus weren't there some bad versions in between? Maybe `>= 40 && < 1` or something?